app/assets/javascripts/plot/highchartsops.coffee

```diff
@@ -92,7 +92,7 @@ class window.HighchartsOps extends PlotOps
     super(resize, reset, registerPen, resetPen, addPoint, updatePenMode, updatePenColor)
     @_chart              = Highcharts.chart(elemID, {})
     @_penNameToSeriesNum = {}
-
+    @_chart.options.exporting.buttons.contextButton.menuItems.pop()
```

Comment:
Ohhh, I didn't realize that the three options were actually different.  The tooltip went off the screen for me, and both `.csv` files and `.xls` files open with the same program on my computer, so they seemed the same.  This is making more sense.  I have way less of a strong preference now, as to whether or not the XLS option should be shown, but I guess we can keep it off.  It seems a little redundant.

But can you make sure to leave a comment there next to the `pop`s, explaining why they're being done?